src/test/e2e.test.ts

```diff
@@ -131,6 +131,19 @@ describe("E2E Private Oracle", () => {
         .wait();
 
       expect(receipt.status).toBe("mined");
+
+      let requesterBalance = await token
+        .withWallet(requester)
+        .methods.balance_of_private(requester.getAddress())
+        .view();
+
+      expect(requesterBalance).toEqual(MINT_AMOUNT - FEE);
+
+      let oracleBalance = await token.methods
+        .balance_of_public(oracle.address)
+        .view();
+
+      expect(oracleBalance).toEqual(FEE);
```

Comment:
Because our tests don't reset the sandbox after each one, can you do oracleBalance = oracleBalanceBefore + FEE so we don't break it by moving the test or adding more on top?

Reply:
I redeploy the token in beforeAll tho (snapshot would be so cool here)